packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -147,8 +143,6 @@ export const VListItem = genericComponent<new () => {
     }))
 
     function onClick (e: MouseEvent) {
-      emit('click', e)
```

Comment:
It needs to emit click on space or enter keydown

Reply:
if add this, should add `emits: ['click']` too, or the event will trigger twice

and if add `emits: ['click']`, `useLink`  will not work well because **Events declared in emits will not exist in attrs**:

```typescript
export function useLink (props: LinkProps, attrs: SetupContext['attrs']): UseLink {
  ...
  const isClickable = computed(() => {
    // Events declared in emits will not exist in attrs
    return isLink?.value || !!(attrs.onClick || attrs.onClickOnce)
  })
  ...
}
```

so if we want to check a bound listener exists,  it needs to be declared in the component's props

```typescript
...
props: ['onClick'],
emits: ['onClick'],
...
```

and need to modify `isClickable` in `useLink` like

```typescript
isLink?.value || !!(attrs.onClick || attrs.onClickOnce || props.onClick || props.onClickOnce)
```

Reply:
Yeah that's probably what I'd go with, we've done that in a couple of other places already. There's a `callEvent` function and `EventProp` interface you can use in `@/util`. I don't know if `@click.once` is passed as a separate prop, you'd have to check that.

Reply:
I notice that EventProp could be an array, dose it mean the listener could be like this?

```vue
<VListItem @click="[foo, bar]" />
```

If it is ,I dont think thats a good way

It will make `props.onClick` be a func like `($event) => [foo, bar]` instead of an array, if the user also passes a similar func, it's hard to tell if the func in the returned array should be executed

If not like this, i had already fixed the issue and could commit right now

Reply:
Yes, that's how events work in vue: 

```js
mergeProps({ onClick: () => {} }, { onClick: () => {} }) === { onClick: [() => {}, () => {}] }
```

If the prop type is just `Function` you'll get runtime errors.